### PR TITLE
Release 2.05.0.0.0 

### PR DESCRIPTION
**Pull request by kantai**
This PR implements SIP-012, and merging to master would release 2.05.0.0.0 into master. See the `CHANGELOG.md` for more information.

- [x] The release must be timed so that it does not interfere with a prepare phase: goal for release of this PR is November 24, which is far from a prepare phase
- [x] Before creating the release, the release manager must determine the version number for this release. This is a feature release, incrementing the feature version number: 2.05.0.0.0
- [x] No outstanding PRs are blockers for this release.
- [x] The release manager must update the CHANGELOG.md file with summaries what was Added, Changed, and Fixed.
- [x] Release candidate tag has been built: https://github.com/blockstack/stacks-blockchain/releases/tag/2.05.0.0.0-rc1
- [x] Once the release candidate has been built, and docker images, etc. are available, the release manager will notify various ecosystem participants to test the release candidate on various staging infrastructure:
   - [x] Stacks Foundation staging environments.
   - [x] Hiro PBC testnet network.
   - [x] Hiro PBC mainnet mock miner.
- [x] A local Radon network test of the 2.05 epoch switch has been performed
- [x] A Xenon network test of the 2.05 epoch switch has been performed
- [x] The release candidate should be announced in the #stacks-core-devs channel in the Stacks Discord.
- [x] The release manager will test that the release candidate successfully syncs with the current chain from genesis.
- [x] Once the final release candidate has rolled out successfully without issue on the above staging infrastructure, the release manager tags 2 additional stacks-blockchain team members to review the release PR.

### CODE REVIEW

**Review by diwakergupta — APPROVED** [review]
🚢 it!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [2.05.0.0.0]
+
+This software update is a consensus changing release and the
+implementation of the proposed cost changes in SIP-012. This release's
```
Comment (gregorycoppola) [inline CHANGELOG.md:11]: does this need a link?

**Review by gregorycoppola — COMMENTED** [review]
Looks good!

Clarification: Did we actually do the Xenon/Radon tests on this PR or on `next-costs`?

File: CHANGELOG.md (hunk 1 above)
Comment (kantai) [reply CHANGELOG.md:11]: There's nothing to link to yet -- SIP-012 is still just a PR.

**Comment by kantai** [comment]
> Clarification: Did we actually do the Xenon/Radon tests on this PR or on next-costs?

It was on `next-costs`, but this PR _is_ `next-costs`

**Review by gregorycoppola — APPROVED** [review]
OK. Sounds good.

Nice work, everyone.

🚢 it

**Review by jcnelson — APPROVED** [review]
LGTM!

**Pull request merged**